ai-builder/labs/README.md

```diff
@@ -27,16 +27,12 @@ For prediction, text classification, and object detection scenarios you will nee
 
 #### Step 1 : Import AIBuilderLabSolution_1_0_0_1 solution to the Dataverse environment
 
-This creates 3 Dataverse entities - Object Detection Product, Health Feedback, and Online Shopping Intent.
-
-- Object Detection Product
-- Health Feedback
-- Online Shopping Intent
-- Travel feedback
+This will create 5 Dataverse tables - Object Detection Product , Health Feedback, Online Shopping Intent, Travel feedback, Expenses
```

Comment:
...create five (5) .... Remove space after object detection product and add a period at the end. I'm assuming the capitalization of the tables is correct, right?